UI_DSM/UI_DSM.Client/Components/App/CommentCreation/CommentCreation.razor

```diff
@@ -0,0 +1,25 @@
+﻿<!--------------------------------------------------------------------------------------------------------
+// CommentCreation.razor
+// Copyright (c) 2022 RHEA System S.A.
+// 
+// Author: Antoine Théate, Sam Gerené, Alex Vorobiev, Alexander van Delft, Martin Risseeuw, Nabil Abbar
+// 
+// This file is part of UI-DSM. 
+// The UI-DSM web application is used to review an ECSS-E-TM-10-25 model.
+// 
+// The UI-DSM application is provided to the community under the Apache License 2.0.
+-------------------------------------------------------------------------------------------------------->
+<EditForm Context="editFormContext" Model="@this.ViewModel.Comment" OnValidSubmit="@this.ViewModel.OnValidSubmit">
+	<DataAnnotationsValidator />
+	<DxFormLayout>
+		<DxFormLayoutItem Caption="Comment content :" ColSpanMd="12">
```

Comment:
I dont think you need any caption here to be honest. The title of the dialog already says "Create Comment", so you can just leave it as a texbox taking full width. But, what could be nice is to make the textbox at least multiline or at best some sort of WYSYWIG editor (perhaps check DevExpress or @omaratrhea).